provisioning/templates-api/kubernetes/templates/api/deployment.yaml

```diff
@@ -45,6 +45,11 @@ spec:
                 configMapKeyRef:
                   name: templates-api
                   key: storageApiHost
+            - name: TEMPLATES_API_PUBLIC_ADDRESS
+              valueFrom:
+                configMapKeyRef:
+                  name: templates-api
+                  key: templatesApiHost
```

Comment:
Ako som vytvaral `config struct` pre kazdu service, tak tu som zabudol pridat ENV.

Kedze sa nemenil provisioning priamo, tak sa k8s testy v CI nespustili, tj. spadlo to az pri release, kde sa k8s test spusta vzdy.